docusaurus/docs/guides/kubernetes/self-hosting/kubernetes-controller.mdx

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 20
+sidebar_label: Ziti Controller
+title: How to Install Ziti Controller in Kubernetes
```

Comment:
Same comment for console and router. I don't see the need for "How to Install " nor "Kubernetes". I know it's kubernetes already.